docs/antrea-network-policy.md

```diff
@@ -1774,3 +1774,11 @@ Similar RBAC is applied to the ClusterGroup resource.
   matched at the same priority with conflicting actions. It will be the policy writer's
   responsibility to identify such ambiguities in rule definitions and avoid potential
   nondeterministic rule enforcement results.
+- Antrea has a higher-priority rule for the established connections, which is expected to take
+  the same "allow" action for the subsequent packets as it did for the first packet.
+  An existing connection was set up before a drop/reject rule was realized, meaning it was
+  expected to "allow" such traffic when the connection was established. So the subsequent valid
+  packets in this connection are also processed with "allow" action.
+  After traffic stops, the existing connection record may be removed from conntrack cache after a
+  timeout interval, and then the drop/reject rule will be applied to the new connection with same
+  attributes.
```

Comment:
I feel this is too implementation specific. Antrea NetworkPolicy is not different from K8s NetworkPolicy on this behavior, and it's also the common behavior across all NetworkPolicy providers. I think most users understand it but if we really want to document it, we could just mention NetworPolicy is connection/flow oriented and stateful. It applies to connection, instead of individual packets, which is easier to understand than explaining the flow priority and conntrack.

Reply:
cc @antoninbas @jianjuns

Reply:
I agree with what Quan said.

Reply:
Thanks. Refactored this part.